### PR TITLE
Update changelog for 1.9.3

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
We should revert https://github.com/microsoft/vscode-cpptools/pull/8820 -- not sure yet if that reversion should be mentioned in the changelog or not.

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+### New Feature
+* Add debugger support for Apple M1 (osx-arm64). [#7035](https://github.com/microsoft/vscode-cpptools/issues/7035)
+   * Resolves issue "[Big Sur M1] ERROR: Unable to start debugging. Unexpected LLDB output from command "-exec-run". process exited with status -1 (attach failed ((os/kern) invalid argument))". [#6779](https://github.com/microsoft/vscode-cpptools/issues/6779)
+
+### Enhancements
+* Add tag parser error logging. [#8907](https://github.com/microsoft/vscode-cpptools/issues/8907)
+* Prevent stripping of format specifiers from -exec commands.
+  * Gareth Rees (@gareth-rees) [MIEngine#1277](https://github.com/microsoft/MIEngine/pull/1278)
+* Improve messages for unknown breakpoints and watchpoints.
+  * Gareth Ree (@gareth-rees) [MIEngine#1282](https://github.com/microsoft/MIEngine/pull/1283)
```
Comment (WardenGnaw) [inline Extension/CHANGELOG.md]:
```suggestion
  * Gareth Rees (@gareth-rees) [MIEngine#1282](https://github.com/microsoft/MIEngine/pull/1283)
```

Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Ooops

**Comment by michelleangela** [comment]
> We should revert #8820 -- not sure yet if that reversion should be mentioned in the changelog or not.

Since releasing the M1 debugger replaces the workaround, then no need to mention it.

**Review by michelleangela — APPROVED** [review]

**Pull request merged**